sap/5.7/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+  .domain.local = DOMAIN.LOCAL
+   domain.local = DOMAIN.LOCAL
+
+[logging]
+    kdc = FILE:/var/log/krb5/krb5kdc.log
+    admin_server = FILE:/var/log/krb5/kadmind.log
+    default = SYSLOG:NOTICE:DAEMON
+
+[libdefaults]
+    default_ccache_name = FILE:/tmp/krb5cc_%{uid}
+----
++
+. Import the key to the `etc` directory.
++
+`ktutil`
++
+`wkt /etc/krb5.keytab`
++
+`q`
++
+. Obtain a Kerberos Ticket Granting Ticket (TGT).
++
+`kinit -V SAPService/sapdev@SSODOMAIN.LAB`
++
+. Change the permissions for the keytab.
++
+`chgrp sapsys /etc/krb5.keytab`
++
+`chmod 640 /etc/krb5.keytab`
++
+. Set up automatic renewal for the Kerberos TGT.
++
+`crontab –e 01 0,6,12,18 * * * /usr/bin/kinit -R SAPService/sapdev@SSODOMAIN.LAB`
++
+. Go to the location of the GSS-API library in the SAP server (in this example, it is under the `/usr/lib64/libgssapi_krb5.so.2` path) and configure the parameters for SNC with Kerberos in the Default Profile and then restart SAP.
+* `snc/permit_insecure_start = 1`
+* `snc/data_protection/use = 3`
+* `snc/data_protection/max = 3`
+* `snc/data_protection/min = 1`
+* `snc/accept_insecure_r3int_rfc = 1`
+* `snc/accept_insecure_rfc = 1`
+* `snc/accept_insecure_cpic = 1`
+* `snc/accept_insecure_gui = 1`
+* `snc/gssapi_lib = /usr/lib64/libgssapi_krb5.so.2`
+* `snc/enable = 1`
+* `snc/identity/as = p:SAPService/sapdev@SSODOMAIN.LAB`
+* `login/password_change_for_SSO = 0`
+. Create `adminsap` as a user in SAP and set the SNC name using the transaction `SU01`.
+. For the RFC destination, use the transaction `SM59`. In the tab logon and security, enable the SNC option and add the SPN name as a partner.
+
+== Configure SAP Connector
+
+Configure SAP Connector to enable SNC with Kerberos:
+
+. Access Anypoint Studio.
+. Select *SAP* in the Studio canvas.
+. Click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. Configure the Kerberos connection:
+.. In the *Connection* field, select `Kerberos`.
+.. On the *General* tab, configure these fields:
+** *SAP system number* 
+** *SAP client ID*
+** *Kerberos Config File Path*
++
+Path to the `krb5.conf` file using the KDC address and the realm information (SSODOMAIN.LAB).
+** *Keytab File Path*
++
+Path to the keytab for the SPN from Active Directory.
+** *GSS Library Path*
++
+Static value `/usr/lib64/libgssapi_krb5.so.2.2` when using CloudHub or static value `/usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2.2` when using CloudHub 2.0, Runtime Fabric, and Mule Runtime On-Premises on Ubuntu. The GSS library is compatible with macOS, Windows, HP-UX, AIX, Solaris, and Red Hat Enterprise Linux. For Mule Runtime On-Premises on macOS, Windows, HP-UX, AIX, Solaris, and Red Hat Enterprise Linux, specify the path to the GSS library installed on the server. For more information about downloading and installing the GSS library, refer to https://www.gnu.org/software/gss/manual/gss.html#Downloading-and-Installing[Downloading and Installing].
```

Comment:
```suggestion
Static value `/usr/lib64/libgssapi_krb5.so.2.2` when using CloudHub or static value `/usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2.2` when using CloudHub 2.0, Runtime Fabric, and Mule runtime engine on-premises on Ubuntu. The GSS library is compatible with macOS, Windows, HP-UX, AIX, Solaris, and Red Hat Enterprise Linux. For Mule on-premises on macOS, Windows, HP-UX, AIX, Solaris, and Red Hat Enterprise Linux, specify the path to the GSS library installed on the server. For more information about downloading and installing the GSS library, refer to https://www.gnu.org/software/gss/manual/gss.html#Downloading-and-Installing[Downloading and Installing].
```

Reply:
For Mule refs, see https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-M